<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationComparison</class>
 <widget class="qMRMLWidget" name="SegmentationComparison">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>704</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>INPUTS</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>CSV file path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="ctkPathLineEdit" name="csvPathSelector">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>ctkPathLineEdit::AdjustToMinimumContentsLength</enum>
          </property>
          <property name="minimumContentsLength">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearCSVPathButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Volume input directory:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkDirectoryButton" name="inputDirectorySelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="caption">
         <string>Select a directory containing the segmentations to compare</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Load volumes from directory:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Load volumes from input folder</string>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="comparisonCollapsibleButton">
     <property name="text">
      <string>COMPARISON</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="controls">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Volume threshold:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QSlider" name="imageThresholdSliderWidget">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>305</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="thresholdPercentageLabel">
             <property name="text">
              <string>100%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="resetCameraButton">
           <property name="text">
            <string>Reset camera</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="survey">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout_9">
         <item row="0" column="0">
          <widget class="QLabel" name="surveyMessage">
           <property name="text">
            <string>Rate the displayed volumes on a scale from 1 to 5:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>LEFT side:</string>
             </property>
             <layout class="QFormLayout" name="formLayout_5">
              <item row="0" column="0" colspan="2">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="leftButtons">
                  <item>
                   <widget class="QRadioButton" name="radioButton_0">
                    <property name="text">
                     <string>1</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">leftGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_4">
                    <property name="text">
                     <string>2</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">leftGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_3">
                    <property name="text">
                     <string>3</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">leftGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_2">
                    <property name="text">
                     <string>4</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">leftGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_1">
                    <property name="text">
                     <string>5</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">leftGroup</string>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>RIGHT side:</string>
             </property>
             <layout class="QFormLayout" name="formLayout_11">
              <item row="0" column="0" colspan="2">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <layout class="QHBoxLayout" name="rightButtons">
                  <item>
                   <widget class="QRadioButton" name="radioButton_5">
                    <property name="text">
                     <string>1</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">rightGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_9">
                    <property name="text">
                     <string>2</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">rightGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_8">
                    <property name="text">
                     <string>3</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">rightGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_7">
                    <property name="text">
                     <string>4</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">rightGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_6">
                    <property name="text">
                     <string>5</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">rightGroup</string>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="save">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Save answers:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="surveyNavigation">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Browse volumes:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="previousButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Previous</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nextButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string/>
             </property>
             <layout class="QFormLayout" name="formLayout_12">
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Total number of comparisons:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="totalComparisonLabel">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string/>
             </property>
             <layout class="QFormLayout" name="formLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Number of comparisons this session:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="sessionComparisonLabel">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="settingsCollapsibleButton">
     <property name="text">
      <string>SETTINGS</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Save path for survey results:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkDirectoryButton" name="outputDirectorySelector"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="leftGroup"/>
  <buttongroup name="rightGroup"/>
 </buttongroups>
</ui>
